test/transaction.ts

```diff
@@ -18,6 +18,7 @@ import * as pfy from '@google-cloud/promisify';
 import * as arrify from 'arrify';
 import * as assert from 'assert';
 import * as proxyquire from 'proxyquire';
+import { configure } from 'protobufjs';
```

Comment:
this gets imported, but doesn't look like you use it

Reply:
Removed!